imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -26,7 +26,7 @@ energy_attrs:
     VALIDMAX: 127
     VAR_TYPE: support_data
 
-epoch:
+epoch_attrs:
```

Comment:
I was messing with this when trying to debug some errors, but decided to keep it because I thought it was more explicit (and thus better). Happy to change it back if you feel strongly about it.

Reply:
Nope, I just like consistency, but it appears we aren't consistent at the moment when looking through other instruments yamls, so no problem choosing one here :)